README.md

```diff
@@ -110,7 +108,7 @@ You can also run `./ora.sh help` to see more commands for administration of the
 
 #### Step 4: Accessing your programming environment
 
-Start your browser at: http://localhost:1999
+Start your browser [here](http://localhost:1999).
```

Comment:
Can you change this to explicitly state `http://localhost:1999`, that information should be included and not hidden by the link.

Reply:
Made the changes.